python/fastquant/backtest/data_prep.py

```diff
@@ -92,6 +106,10 @@ def include_sentiment_score(data, sentiments):
         senti_series, left_index=True, right_index=True, how="left"
     )
     data = data.reset_index()
+    # dividend handler in sentiment strat
+    if "dividend" not in data.columns:
+        data["dividend"] = 0
```

Comment:
Ah, so this should make it not an issue if a dividend column doesn't exist ...

So wondering why there was an issue.

Reply:
Ahhh because it's only reflected in the sentiment strat, this should be applied for all cases.